examples/demo_1d_swe.py

```diff
@@ -0,0 +1,52 @@
+
+from __future__ import division  # Get proper divison
+
+import math
+import random
+
+import numpy as np
+
+
+from firedrake import *
+
+from flooddrake import *
+
+""" demo file for simple 2d shallow water equations """
```

Comment:
Put file docstring at the beginning of the file. Trim leading and trailing whitespace in docstring.